tests/functional/timestamps/test_timestamps.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+from dbt.tests.util import check_relation_has_expected_schema, run_dbt
+
+_MODEL_CURRENT_TIMESTAMP = """
+SELECT {{current_timestamp()}} as current_timestamp,
+       {{current_timestamp_in_utc()}} as current_timestamp_in_utc
+"""
+
+
+class TestCurrentTimestamps:
+    @pytest.fixture(scope="class")
+    def models(self):
+        return {"get_current_timestamp.sql": _MODEL_CURRENT_TIMESTAMP}
+
+    def test_current_timestamps(self, project, models):
+        results = run_dbt(["run"])
+        assert len(results) == 1
+        check_relation_has_expected_schema(
+            project.adapter,
+            relation_name="get_current_timestamp",
+            expected_schema={
+                "current_timestamp": "timestamp with time zone",
+                "current_timestamp_in_utc": "timestamp without time zone",
+            },
+        )
```

Comment:
This looks like a common way that schema comparisons are accomplished currently within pytests:
https://github.com/dbt-labs/dbt-core/blob/3bfce2bac91a89a2886785e2fb31315e41830239/tests/adapter/dbt/tests/adapter/utils/data_types/base_data_type_macro.py#L26-L34

It uses this function:
https://github.com/dbt-labs/dbt-core/blob/0bd6df0d1b1e4e382c413929403f7972108a5b8a/core/dbt/tests/util.py#L447-L453

Reply:
yea I saw this but it means I need to create a comparison table which feels unnecessary and less explicit